theme/templates/pages/homepage.html

```diff
@@ -196,6 +196,7 @@ <h4>Permanently Publish</h4>
 {% endblock %}
 
 {% block extra_head %}
+{% if not page|is_debug %}
```

Comment:
is this so crawlers don't pick it up or something?

Reply:
Exactly, we don't want crawlers to pick up things on our testing and staging machines.